app.config.js

```diff
@@ -93,6 +93,30 @@ module.exports = function (config) {
           NSPhotoLibraryUsageDescription:
             'Used for profile pictures, posts, and other kinds of content',
           CFBundleSpokenName: 'Blue Sky',
+          CFBundleLocalizations: [
```

Comment:
I don't quite follow?

Reply:
I think @gaearon may refer to the fact that it would be even better to automatically generate the list of localizations based on `Object.keys(APP_LANGUAGES)` (`APP_LANGUAGES` being imported from `src/locale/languages.ts`)!

Reply:
Ah I see. That might indeed be better, I wouldn't have a clue how to do it though, I'm afraid :)

Reply:
To be fair, it would be quite hard to do as-is, as the `app.config.js` is written in JS, and we would need to rewrite it in Typescript as `app.config.ts` (and make sure the toolchain uses `ts-node` according to [Expo's documentation](https://docs.expo.dev/workflow/configuration/#dynamic-configuration)) to be able to import `APP_LANGUAGES` here.

I recommand to keep the automatisation for a follow-up pull request by a more seasoned developer 😅

Reply:
Sounds fine to me! :)